exchanges/coinbasepro/coinbasepro_test.go

```diff
@@ -1659,17 +1660,30 @@ func TestProcessSnapshotUpdate(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-func TestGenerateDefaultSubscriptions(t *testing.T) {
-	comparison := subscription.List{{Channel: "heartbeats"}, {Channel: "status"}, {Channel: "ticker"},
-		{Channel: "ticker_batch"}, {Channel: "candles"}, {Channel: "market_trades"}, {Channel: "level2"}}
-	for i := range comparison {
-		comparison[i].Pairs = currency.Pairs{
-			currency.NewPairWithDelimiter(testCrypto.String(), testFiat.String(), "-")}
-		comparison[i].Asset = asset.Spot
+func TestGenerateSubscriptions(t *testing.T) {
+	t.Parallel()
```

Comment:
If I understand correctly, since this test is surrounded by others without t.Parallel() at the start, it won't actually run in parallel with any of them. This line should either be removed, or the test should be relocated, for the desired behaviour.

Reply:
No, I don't think that's the correct way to view Parallel tests.
* They don't have to live next to other parallel tests. There's no relationship that parallel tests should go anywhere in the test file, or be colocated.
* Our default should be for nearly all tests to be parallel. 
But me making this one parallel doesn't mean I'm taking responsibility for fixing all other tests. In normal cases I might actually do that, but in this case I'm PR'ing into *your* PR, so I don't want to introduce more changes than necessary